doc/glossary.rst

```diff
@@ -369,14 +369,6 @@ general neuroimaging concepts. If you think a term is missing, please consider
         See :class:`SourceSpaces` for the class definition.
 
     stim channel
-        See :term:`trigger channel`.
```

Comment:
@hoechenberger such cross-references are unnecessary; sphinx glossary allows multiple terms to have the same definition (see, e.g., `dipole, ECD, equivalent current dipole`) and for alphabetization of the entries the first term given is the one considered.